ttnn/cpp/ttnn/device_operation.hpp

```diff
@@ -332,6 +340,12 @@ void launch_on_worker_thread(auto cq_id, auto operation_id, const auto& operatio
             },
             program);
 
+            for (auto& cb : program_ptr->circular_buffers()) {
+                tt::tt_metal::GraphTracker::instance().track_allocate_cb(cb->core_ranges(), 0, cb->size());
+            }
```

Comment:
This looks like a complete copy-pasta between one branch and another. 
```
        if (USE_FAST_DISPATCH) {
            ZoneScopedN("EnqueueProgram");
            auto& queue = device->command_queue(cq_id);
            tt::tt_metal::EnqueueProgram(queue, program_ptr, false);
        } else {
            ZoneScopedN("LaunchProgram");
            ::detail::LaunchProgram(device, program_ptr);
        }
        TracyOpTTNNDevice(
            device_operation_t{},
            operation_id,
            device->id(),
            *program_ptr,
            operation_attributes,
            tensor_args,
            tensor_return_value);
```

same with 
```
            for (auto& cb : program_ptr->circular_buffers()) {
                tt::tt_metal::GraphTracker::instance().track_allocate_cb(cb->core_ranges(), 0, cb->size());
            }
```

Maybe there a chance to remove duplication.

Reply:
And I think we should remove USE_FAST_DISPATCH this branch, as discussed yesterday 😅
Not sure if in this PR or not though